networking/multiple_networks/configuring-additional-network.adoc

```diff
@@ -187,46 +187,52 @@ spec:
   desiredState:
     ovn:
       bridge-mappings:
-      - localnet: physnet1 <3>
+      - localnet: localnet1 <3>
         bridge: br-ex <4>
         state: present <5>
 ----
 <1> The name for the configuration object.
 <2> A node selector that specifies the nodes to apply the node network configuration policy to.
-<3> The name for the network interface.
-<4> The name of the bridge on the node. This value is required only if you specify `state: present`.
+<3> The name for the additional network from which traffic is forwarded to the OVS bridge. This additional network must match the name of the additional network that you define in the `netAttachDefName` field of a `NetworkAttachmentDefinition` object.
```

Comment:
this is wrong; it must match - verbatim - the value of the `name` field of a `NetworkAttachmentDefinition` object instead. Not the `netAttachDefName`.